examples/bigquery-table-access-pattern-analysis/src/bq_name.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019-2020 Faculty Science Limited
```

Comment:
Why is it Faculty Science Limited?

Reply:
meanwhile double checking on all the copyright license on each files. Please update the year to 2021 to give accurate information